vmdb/app/models/host.rb

```diff
@@ -1244,15 +1244,21 @@ def detect_discovered_os(ost)
   def detect_discovered_hypervisor(ost, ipaddr)
     log_header = 'MIQ(Host.detect_discovered_hypervisor)'
     find_method = :find_by_ipaddress
+
+    # TODO: break this apart, we probably want to only do the namelookup for hyperv, esx
+    hostname = Socket.getaddrinfo(ipaddr, nil)[0][2]
```

Comment:
If you spell it correctly, sure. :trollface: 

All joking aside, I didn't know about `Resolv` class (why is it Resolv and not Resolve, btw?  I hate random abbreviations).  If that works, we should kill our MiqSockUtil method.